README.md

```diff
@@ -8,7 +16,7 @@
 
 ## Introduction
 
-You are tasked with building a small frontend application containing 3 forms and a list. There is a screenshot at the bottom of this document that gives you an idea of what to aim for. As you'll be able to see, it doesn't have to *look* good so don't spend time on styling!
+You are tasked with building a small frontend application containing 3 forms and a list. There is a screenshot at the bottom of this document that gives you an idea of what to aim for. As you'll be able to see, it doesn't have to _look_ good so don't spend time on styling!
```

Comment:
is prettier changing this or did you do it manually? .....

Reply:
KSDKSAJDKJDKASJDKASDK prettier 😂